models/models_test.go

```diff
@@ -191,6 +192,15 @@ func TestValidateCodelist(t *testing.T) {
 			So(invalidFields, ShouldBeNil)
 		})
 
+		Convey("when isCantabularGeography is missing", func() {
+			codelist := createCodeList()
+			codelist.IsCantabularGeography = nil
+			missingFields, invalidFields := codelist.validateCodelist(ctx)
```

Comment:
Not a problem for this PR, but a note for future: It'd be nice if these methods didn't include so much stutter. codelist.validate() would be just as clear. I know there are two validate() methods in models.go but because they take different receiver types (instances & codelists) we should be able to distinguish them by that context.

If you have any similar work in future it might be worth factoring the name change in :)